src/tests/Common/CLRTest.CrossGen.targets

```diff
@@ -285,7 +299,9 @@ if defined RunCrossGen2 (
     Exit /b 0
 
 :DoneCrossgen2Operations
+    echo Crossgen2 compilation finished, exit code !CrossGen2Status!>>!compilationDoneFlagFile!
     call :ReleaseLock
+:DoneCrossgen2OperationsNoRelease
```

Comment:
Nit: This label looks unused?

Reply:
For sure, that's because I messed up the waiting loop. Thanks for pointing that out!